README.md

```diff
@@ -1 +1,184 @@
+
+### Key Features <a name="key-features"></a>
+
+
+- Added HTML 
+- Added CSS 
+- Added javascript 
+- Added webpack as module bundler 
+- fetch data from real API 
+
+
+### Testing <a name="Testing"></a>
+
+- we used Jest as testing library in this project 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## live demo <a name="livelink">Coming Soon</a>
+- [clickhere](https://zuwairanajma.github.io/Javascript-Capstone_AlbertandJuw/dist)
+- [video](https://drive.google.com/file/d/1EyeCIsYWUFGKX8nFjz2ayp_4T8OqxJGL/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1: web browser
+2: a code editor
+3: node installed on your pc
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd to your desired folder 
+  git clone https://github.com/Zuwairanajma/Javascript-Capstone_AlbertandJuw.git
+### Install
+
+Install this project with:
+  install node_modules by this command : npm i -y
+
+then:
+
+run this command: npm run start 
+
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Zuwaira Sadiq**
+- GitHub: [@Zuwairanajma](https://github.com/Zuwairanajma)
+- Twitter: [@SadiqJuwairiyya](https://twitter.com/SadiqJuwairiyya)
+- LinkedIn: [Zuwaira Sadiq](https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?)
+
+
+👤 **Albert Antwi**
+- GitHub: [@albertkantwi](https://github.com/albertkantwi)
+- Twitter: [@Albert_KAntwi](https://twitter.com/Albert_KAntwi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/albert-antwi-557291263/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Functionalities**
+- [ ] **Improved UI design**
+- [ ] **Enhance Designs for all screen sizes and other media queries such as "print**
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project don't forgt to give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microvesre who helped us during our journey
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] As suggested by the previous reviewer please fix the MIT License link as its broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.